roc/version_test.go

```diff
@@ -80,3 +81,199 @@ func TestVersion_Parse(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion_Validite(t *testing.T) {
+	tests := []struct {
+		name    string
+		version Versions
+		wantErr error
+	}{
+		{
+			name: "compatible: equal versions",
+			version: Versions{
+				Native: SemanticVersion{
+					Major: 1,
+					Minor: 1,
+					Patch: 1,
+				},
+				Bindings: SemanticVersion{
+					Major: 1,
+					Minor: 1,
+					Patch: 1,
+				},
```

Comment:
nit: maybe tests will be more compact & readable if we omit fields for SemanticVersion? Like:

```go
				Native: SemanticVersion{1, 1, 1},
				Bindings: SemanticVersion{1, 1, 1},
```